packages/help-center/src/styles.scss

```
@@ -169,7 +169,7 @@ $head-foot-height: 50px;
 			}
 		}
 
-		.components-button {
+		components-button:not(.help-center-contact-form__site-picker-cta) {
```

```suggestion
		.components-button:not(.help-center-contact-form__site-picker-cta) {
```

Does this need to be a class?